pkgs/top-level/perl-packages.nix

```
@@ -16716,6 +16716,21 @@ let
     };
   };
 
+  Rainbarf = buildPerlModule {
+    pname = "Rainbarf";
```

```suggestion
  Apprainbarf = buildPerlModule {
    pname = "Apprainbarf";
```
The attribute name and pname should follow the conventions. Also use `perlPackages.Apprainbarf` in the commit message and PR title.